tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -70,47 +113,75 @@ def test_config_cli_priority(convert2rhel):
 
     assert c2r.exitstatus == 1
 
-    remove_files(config)
-
-
-def test_config_standard_paths_priority_diff_methods(convert2rhel):
-    config = [
-        Config("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
-        Config("/etc/convert2rhel.ini", "[subscription_manager]\nactivation_key = config2_activationkey"),
-    ]
-    create_files(config)
 
+@pytest.mark.parametrize(
+    "set_up_config_files",
+    [
+        [
+            Config(
+                "~/.convert2rhel.ini",
```

Comment:
Just to be sure, I would put an exact path
```suggestion
                "/root/.convert2rhel.ini",
```